packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -1114,5 +1114,23 @@ export const Rules: { [id in Payloads.PayloadType]: Rule } = {
 				return null;
 			}
 		}
+	},
+
+	// ========================================================================
+	'x-reserve-actor': {
```

Comment:
>'x-reserve-actor' [](start = 1, length = 17)

I get what you are doing here in trying to name this and identify it as a exception type, but maybe we could be more descriptive here.  How about 'session.reserve-actor'?

Reply:
My take: I like this name. It's not overly specific, it's inline with custom http header naming conventions, and alphabetically sorts toward the end of a list.

---
In reply to: [317267483](https://github.com/microsoft/mixed-reality-extension-sdk/pull/423#discussion_r317267483) [](ancestors = 317267483)